dashboard/app/controllers/courses_controller.rb

```diff
@@ -8,9 +10,16 @@ def index
   end
 
   def show
-    course_name = params[:course_name].tr('-', '_').titleize
-    course = Course.find_by_name(course_name)
-    raise ActiveRecord::RecordNotFound unless course
+    course = Course.find_by_name(params[:course_name])
+    unless course
+      # PLC courses have different ways of getting to name. ideally this goes
+      # away eventually
+      course_name = params[:course_name].tr('-', '_').titleize
+      course = Course.find_by_name!(course_name)
+      # only support this alternative course name for plc courses
+      raise ActiveRecord::RecordNotFound unless course.plc_course
```

Comment:
`find_by_name!` raises if the course is not found, but we also want to raise if we found a course, but it is a regular course rather than a plc_course